code/game/machinery/cell_charger.dm

```diff
@@ -91,7 +91,7 @@
 		if(!charging || (stat & (BROKEN|NOPOWER)) || !anchored)
 			return
 
-		use_power(2000)		//this used to use CELLRATE, but CELLRATE is fucking awful. feel free to fix this properly!
-		charging.give(1750)	//inefficiency.
+		use_power(200)		//this used to use CELLRATE, but CELLRATE is fucking awful. feel free to fix this properly!
+		charging.give(175)	//inefficiency.

```

Comment:
I know it's outside of these changes. Would this be the file to work on to put a stop to the circular recharging? As it stands right now, you can keep switching APC powercells and recharging them on rechargers. They really should spike the hell out of the APC and have power efficiency losses.